src/pages/VenuePage/VenuePage.tsx

```diff
@@ -19,6 +19,7 @@ import {
   WorldWithId,
 } from "types/id";
 import { Profile, UserLocation } from "types/User";
+import { BaseVenue } from "types/venues";
```

Comment:
Probably `SpaceWithId` would be better fit. Noted the reason in a comment before.

Reply:
Agreed. Fixed in 22c9bf0fd